src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/entities/BailCaseFieldDefinition.java

```diff
@@ -222,6 +222,8 @@
         "hasLegalRep", new TypeReference<YesOrNo>(){}),
     HEARING_CENTRE(
         "hearingCentre", new TypeReference<HearingCentre>(){}),
+    DETENTION_FACILITY_NAME(
+        "detentionFacility", new TypeReference<String>(){}),
```

Comment:
Can we please try to follow a convention -> DETENTION_FACILITY : detentionFacility;
UPPER_UNDERSCORE : lowerCamelCase.
It will be easier for tests and later we need to make these changes in the documents_api as well.

Reply:
Refactored everything there now thanks